packages/office-addin-usage-data/src/usageData.ts

```diff
@@ -282,13 +282,14 @@ export class OfficeAddinUsageData {
    */
   public maskFilePaths(err: Error): Error {
     try {
+      const error: Error = err;
```

Comment:
This doesn't work.  Note the check-in I made recently to reportException to clone the object . . . that's because statement like this end up just be a new reference to the same data.

Reply:
We definitely don't want the side effects. I was having a problem with it when reporting an exception and testing the error value in the manifest (because reporting the exception changed the error).

You are right that simply assigning the error to a new variable doesn't solve the problem.
This lead me to that the tests here are broken in which the side effects are making one test depend on another. Working on it.

Reply:
@millerds you are right that didn't fix it. Those side effects were making the tests pass wrongfully (one test was depending on others). Did some work to fix these errors as well.
I also moved the side effect handling to the report error functions as I also think it makes sense for the mask function to change the error.